packages/docs/pages/users/wallet/web-wallet.mdx

```diff
@@ -45,34 +52,34 @@ Remember: there's no 'recover password with email' option!
 Your wallet is ready to use.
 </Steps>
 
-## Using the Extension
-The extension itself only provides key management functionality. To send or receive tokens, you will need to navigate to a compatible
-web app (such as [namada-interface](https://github.com/anoma/namada-interface)) and connect the extension to the site.  
+## Using Namada Keychain
+The Namada Keychain extension itself only provides key management functionality. To send or receive tokens, you will need to navigate to a compatible
+web app (such as [Namadillo](https://github.com/anoma/namada-interface)) and approve a connection to Namada Keychain from the site.  
 
-Here's how to use the extension with Namada-Interface:
+Here's how to use Namada Keychain with Namadillo:
 
 <Steps>
-### Set the network in the extension
-Open the extension and click the 'gear' icon in the top right, then select 'Network'.  
+### Set the network in Namada Keychain
+Open Namada Keychain and click the 'gear' icon in the top right, then select 'Network'.  
 
 Enter a valid chain-id and rpc url for the chain you wish to transact on, and click 'Submit'.
 
-### Navigate to the Namada-interface site
-The Namada-interface for the [Campfire testnet](../../networks/testnets/campfire.mdx) is hosted at [https://interface.luminara.icu](https://interface.luminara.icu). After mainnet
-has launched, a list of hosted Interfaces will be found [here](../../networks/mainnets.mdx).
+### Navigate to a hosted instance of Namadillo
+The Namadillo instance for the [Campfire testnet](../../networks/testnets/campfire.mdx) is hosted at [https://interface.luminara.icu](https://interface.luminara.icu). After mainnet
+has launched, a list of hosted Namadillo instances will be found [here](../../networks/mainnets.mdx).
 
-### Connect the extension
-Click 'Connect to Namada Extension' and approve when prompted. You should see your wallet balance displayed on the page.
+### Connect Namada Keychain
+Click the 'Connect Extension' button and approve when prompted. If everything is configured correctly, you should now see your wallet balance displayed on the page.
 
 ### Interact
-Once connected, you can use the Interface to make transfers or stake with a validator (Note: some functionality may be missing on testnet).
-Whenever you submit a transaction to the chain, you will be prompted to enter your password to allow the extension to sign using your private key.
+Once connected, you can use Namadillo to make transfers or stake with a validator (Note: some functionality may be missing on testnet).
```

Comment:
This is mostly correct, but transfers aren't enabled currently.

Reply:
Does the transfer functionality generally exist though?

Reply:
Not currently. We don't have the UI for this currently. Phase 3 will introduce the IBC & Shielded Transfer views, which are being developed right now!